app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> int:
+        key = (func.__name__, args, frozenset(kwargs.items()))
```

Comment:
In this case, it is not necessary to store the function name as a key; you can cache only the args. ( `cache_dict[args]` )

Reply:
I know I don't need a func __name__, but I wanted to improve it a little)

Reply:
If you want ) Okay,  let it be here : )